### PR TITLE
Features/268 app layout menubar navigation

### PR DESCRIPTION
**Pull request by Horizon19911**
@oaksoe  @minkhant285

### CODE REVIEW

File: src/ultis/YupValidation.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as Yup from 'yup';
```
Comment (minkhant285) [inline src/ultis/YupValidation.tsx]: yarn add @types/yup

File: src/components/Login/login.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import { useState,useContext } from 'react';
+import { styles } from './styles'
+import { View,Text,TextInput,TouchableOpacity } from 'react-native';
+import { compose } from 'recompose';
+import { Formik, FormikProps } from 'formik';
+import { handleTextInput,withNextInputAutoFocusInput,withNextInputAutoFocusForm } from 'react-native-formik';
+
+import { AuthNavProps } from 'src/ultis/ParamLists/AuthParamList';
+
+import { LoginSchema } from '../../ultis/YupValidation'
```
Comment (minkhant285) [inline src/components/Login/login.tsx]: move utils folder under common and move all common use files to under common folder

File: src/Tabs/AppTabs.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { useContext } from "react";
```
Comment (minkhant285) [inline src/Tabs/AppTabs.tsx]: remove useContext if not use

File: src/Routes/index.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+interface RoutesProps {}
+
+export const Routes: React.FC<RoutesProps> = ({}) => {
+  const { token } = useContext(AuthContext);
+  const [loading, setLoading] = useState(true);
+  const [isLogined,setLogin] = useState(false);
+
+  useEffect(() => {
+    AsyncStorage.getItem("devx_token")
+      .then(localToken => {
```
Comment (minkhant285) [inline src/Routes/index.tsx:23]: change to (loacalToken : any)

File: src/Routes/index.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        console.log(localToken);
```
Comment (minkhant285) [inline src/Routes/index.tsx:26]: comment this log message

**Review by minkhant285 — CHANGES_REQUESTED** [review]

File: src/Routes/index.tsx (hunk 4 above)
Comment (Horizon19911) [reply src/Routes/index.tsx:23]: bro, can't change to type any

File: src/ultis/YupValidation.tsx (hunk 1 above)
Comment (Horizon19911) [reply src/ultis/YupValidation.tsx]: bro, it's already add
When I change to `import * as Yup from '@type/yup' 
It shows error

**Review by minkhant285 — APPROVED** [review]

**Pull request merged**